source/appModules/calculator.py

```diff
@@ -66,28 +66,25 @@ def event_nameChange(self, obj, nextHandler):
 		nextHandler()
 
 	def event_UIA_notification(self, obj, nextHandler, displayString=None, activityId=None, **kwargs):
-		# #12268: for "DisplayUpdated", announce display strings in braille and move on.
+		calculatorVersion = int(self.productVersion.split(".")[0])
+		# #12268: for "DisplayUpdated", announce display strings in braille  no matter what they are.
```

Comment:
In that case, @josephsl could you add another comment below this one to say that the only type of activity that should be handled here is "DisplayUpdated". Perhaps an example of other activities that shouldn't be handled, or an explanation about why other activities are not relevant.